cypress/e2e/releases/members.spec.js

```diff
@@ -62,14 +64,11 @@ describe("Release > members > CRUD", () => {
         pathname: "/v1/releases/27-d0d8f7a0f8-1pp779/members/*",
       },
       () => {
-        cy.get(
-          "[data-cy=PlanTable] tbody tr:last-child [data-cy=release-member-version] .Select .options li:last-child"
-        ).click();
+        cy.get("[role=presentation] li:last-child").click();
```

Comment:
why is this being changed to a less specific selector?

Reply:
Because the popover is rendered on a portal it no longer follows that dom tree selector